.github/workflows/add-unanswered-to-project.yml

```diff
@@ -0,0 +1,76 @@
+name: Add Unanswered PRs and Issues to PyTorch Org Project 136
```

Comment:
What do you think about renaming to "Add External PRs and Issues...", since we're not filtering by unanswered yet?